Source/santad/SNTNotificationQueue.mm

```diff
@@ -15,6 +15,7 @@
 
 #import "Source/santad/SNTNotificationQueue.h"
 
+#include <Foundation/Foundation.h>
```

Comment:
I'm gonna assume VSCode did this? It's unnecessary, it's imported in the header file

Reply:
Yeah, i've not yet decided if I'm happy with clangd auto includes or not. Sometimes it's nice :/